.github/workflows/sentry-and-heroku-deploy.yml

```diff
@@ -0,0 +1,40 @@
+name: New Sentry Release and Deploy to Heroku
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release:
+   runs-on: ubuntu-18.04
+   env:
+     HEROKU_API_KEY: ${{ secrets.HEROKU_API_KEY }}
+     SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+     SENTRY_DSN: ${{ secrets.SENTRY_DSN }} 
+     SENTRY_ORG: ${{ secrets.SENTRY_ORG }}
+     SENTRY_PROJECT: ${{ secrets.SENTRY_PROJECT }}
+     SENTRY_DEPLOY_ENVIRONMENT: ${{ secrets.SENTRY_DEPLOY_ENVIRONMENT }}
+
+   steps: 
+    # See: https://help.github.com/en/actions/automating-your-workflow-with-github-actions/configuring-a-workflow#using-the-checkout-action
+    - uses: actions/checkout@v1.0.0
+
+    - name: Install dependencies, install Sentry CLI
+      run: |
+        npm install
+        curl -sL https://sentry.io/get-cli/ | bash
+
+    - name: Create new release and new deploy with Sentry CLI
+      run: |
+        SENTRY_RELEASE=$(sentry-cli releases propose-version)
+        sentry-cli releases new -p $SENTRY_PROJECT $SENTRY_RELEASE
+        sentry-cli releases finalize $SENTRY_RELEASE
+        sentry-cli releases set-commits --auto $SENTRY_RELEASE
+        sentry-cli releases deploys $SENTRY_RELEASE new -e $SENTRY_DEPLOY_ENVIRONMENT
```

Comment:
@LearningNerd This looks good to me but I recommend getting a sales engineer to review this as well, ideally Neil if he has time.

Reply:
👍 will do :)

Reply:
this looks good! i think finalize should go after set-commits.

Reply:
Thanks @ndmanvar ! I switched that and fixed it up a bit.